### DISCUSSION TITLE
ar-scale="auto" not working when ar-placement="wall"

### DISCUSSION THREAD

**Discussion by mohanyadav** [google/model-viewer#3640] [Q&A]
### Description

We are using model-viewer to place an object in vertical mode using ar-placement="wall". Also setting the ar-scale="auto" and ar-modes="scene-viewer webxr". But when we tested it we are unable to scale the object by pinch.

We also tested by giving WebXR priority by setting ar-modes="webxr scene-viewer" and now we are able to scale it. But we really like the UX (guides and movable buttons) in scene-viewer and want to give it priority over WebXR.

#### Live Demo
Scaling Not Working Version (Scene-Viewer)- https://caring-prism-closet.glitch.me
Scaling Working Version (WebXR)- https://aquatic-berry-hedgehog.glitch.me

### Version
- model-viewer: v1.5.0

### Browser Affected

- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
@tpsiaki Can SceneViewer scale objects that are in vertical placement mode? This doesn't look like my bug, but I could be wrong.

**Comment by mohanyadav** [comment]
I have tested this separately with only the Scene Viewer parameters and removing model-viewer. In the demo attached below clicking on the Open Sceneviewer button gives us similar results (scaling not working)

I used the following parameters in the demo below -

- file=https://cdn.glitch.com/36cb8393-65c6-408d-a538-055ada20431b/Astronaut.glb?1542147958948
- mode=ar_only
- enable_vertical_placement=true
- resizable=true

Referenced these parameters directly from Sceneviewer docs - https://developers.google.com/ar/develop/java/scene-viewer

Demo Link - https://sable-spurious-adasaurus.glitch.me
